src/main/java/hyperledger/besu/java/rest/client/listener/TransactionConsumer.java

```diff
@@ -83,21 +102,27 @@ public void listen(ConsumerRecord<String, String> message, Acknowledgment acknow
         }
       }
 
-      if (!contractAddress.isEmpty()
+      if (!smartContractAddress.isEmpty()
           && !transactionFunctionName.isEmpty()
-          && !transactionParams.isEmpty()) {
+          && !transactionParameters.isEmpty()) {
         transactionService.execute(
-            abiDefinitionFile, contractAddress, transactionFunctionName, transactionParams);
+            abiDefinitionFile,
+            smartContractAddress,
+            transactionFunctionName,
+            transactionParameters);
       }
       acknowledgment.acknowledge();
-    } catch (BesuTransactionException e) {
+    } catch (BesuTransactionException exception) {
       acknowledgment.acknowledge();
       eventPublishServiceImpl.publishTransactionFailureEvent(
-          e.getMessage(), contractAddress, transactionFunctionName, transactionParams);
-      log.error("Error in Submitting Transaction - Exception - {}", e.getMessage());
-    } catch (Exception ex) {
-      log.error("Error in Kafka Listener - Message Format exception - {}", ex.getMessage());
-      ex.printStackTrace();
+          exception.getMessage(),
+          smartContractAddress,
+          transactionFunctionName,
+          transactionParameters);
+      log.error("Error in Submitting Transaction - {}", exception.getMessage());
+    } catch (Exception e) {
+      log.error("Error in Kafka Listener - Message Format {}", e.getMessage());
+      e.printStackTrace();
```

Comment:
Q: Why printStackTrace ?

Reply:
it was already there. I formatted the line so its showing in the changes